dash-renderer/src/actions/setAppReadyState.js

```diff
@@ -0,0 +1,80 @@
+import {contains, filter, type} from 'ramda';
+import {createAction} from 'redux-actions';
+
+import Registry from './../registry';
+import {getAction} from './constants';
+
+export const SIMPLE_COMPONENT_TYPES = ['String', 'Number', 'Null', 'Boolean'];
+export const isSimpleComponent = component =>
+    contains(type(component), SIMPLE_COMPONENT_TYPES);
```

Comment:
~`contains`~ -> `includes` https://github.com/ramda/ramda/pull/2452

Do the `export`s here imply you're planning to share this with `TreeContainer`?

Reply:
https://github.com/plotly/dash/pull/899/commits/f32edbd8b220e32d895f9a611e24abc5642523f2